<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_search">
          <property name="text">
           <string>Поиск</string>
          </property>
          <property name="icon">
           <iconset resource="my_resources.qrc">
            <normaloff>:/ic0/search_find.ico</normaloff>:/ic0/search_find.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_search"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_connectDB"/>
    <addaction name="action_add"/>
    <addaction name="action_edit"/>
    <addaction name="action_delete"/>
    <addaction name="actionTest"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action_settings_save"/>
    <addaction name="action_settings_reset"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Язык</string>
    </property>
    <addaction name="action_language_English"/>
    <addaction name="action_language_Russian"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_connectDB"/>
   <addaction name="action_add"/>
   <addaction name="action_edit"/>
   <addaction name="action_delete"/>
   <addaction name="actionTest"/>
  </widget>
  <action name="action_connectDB">
   <property name="icon">
    <iconset resource="my_resources.qrc">
     <normaloff>:/ic0/globe.ico</normaloff>
     <normalon>:/ic0/globe.ico</normalon>:/ic0/globe.ico</iconset>
   </property>
   <property name="text">
    <string>Подключение к базе данных</string>
   </property>
  </action>
  <action name="action_add">
   <property name="icon">
    <iconset resource="my_resources.qrc">
     <normaloff>:/ic0/plus_add.ico</normaloff>:/ic0/plus_add.ico</iconset>
   </property>
   <property name="text">
    <string>Добавление записи</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="icon">
    <iconset resource="my_resources.qrc">
     <normaloff>:/ic0/edit_file.ico</normaloff>:/ic0/edit_file.ico</iconset>
   </property>
   <property name="text">
    <string>Редактирование записи</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="my_resources.qrc">
     <normaloff>:/ic0/delete_trash.ico</normaloff>:/ic0/delete_trash.ico</iconset>
   </property>
   <property name="text">
    <string>Удаление записи</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Тест функций</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="my_resources.qrc">
     <normaloff>:/ic0/about.ico</normaloff>:/ic0/about.ico</iconset>
   </property>
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="action_settings_save">
   <property name="icon">
    <iconset resource="my_resources.qrc">
     <normaloff>:/ic0/set.ico</normaloff>:/ic0/set.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранение настроек</string>
   </property>
  </action>
  <action name="action_settings_reset">
   <property name="icon">
    <iconset resource="my_resources.qrc">
     <normaloff>:/ic0/reset.ico</normaloff>:/ic0/reset.ico</iconset>
   </property>
   <property name="text">
    <string>Сброс настроек</string>
   </property>
  </action>
  <action name="action_language_English">
   <property name="icon">
    <iconset resource="my_resources.qrc">
     <normaloff>:/ic0/en.png</normaloff>:/ic0/en.png</iconset>
   </property>
   <property name="text">
    <string>Английский</string>
   </property>
  </action>
  <action name="action_language_Russian">
   <property name="icon">
    <iconset resource="my_resources.qrc">
     <normaloff>:/ic0/ru.png</normaloff>:/ic0/ru.png</iconset>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="my_resources.qrc"/>
 </resources>
 <connections/>
</ui>
